comm_norms.adoc

```diff
@@ -234,104 +282,138 @@ Remember to be gracious and thankful in your discussions with other project part
+
 If an issue has become especially contentious and discourse has become rude or inflammatory, restrict access to that issue for a stated period of time (say 24 to 48 hours) to allow people time to calm down, reflect, and state their argument in a more even-tempered and constructive manner.
 
 === Quick tips for filing issues well
+PD--The bullets below are pushing the limit on length.
 
-. Thank the people creating the software for their time and energy, especially if you are brand new to the project.
+. Thank the people creating the software for their time and energy, especially if you are new to the project.
 The individuals spending their (free) time creating free and open source software for you to use are also people who want to know their time is valued and their work appreciated.
+PD--Maybe "spare" instead of "free"?
+
 . Include as much information as you possibly can about the error you have encountered.
 If the project uses issue templates, fill it out as completely as possible.
-If you do not have the information requested or cannot determine how to get it yourself, then simply note what you have attempted to do in order to get the information.
+PD--There's an issue with number here. You've got templates (plural) and it (singular).
+
+If you do not have the information requested or cannot determine how to get it yourself, simply note what you have attempted to do in order to get the information.
 These details help mainters determine what they might need to do to assist you.
+PD--Did you mean "maintainers" here?
+
 . If a project does not use issue templates, look at other issues that have been "closed‒fixed" or at merged pull requests to see how other people have filed bug reports.
+PD--Make sure the number for "templates" here agrees with whatever you decide in the bullet above.
+
 If the issue was fixed, chances are quite good that you'll be able to use these historical artifacts as examples of the sort of information necessary for reproducing an error.
 Replicate what you find in these reports and add more detail as you are able.
 
 === Quick tips for responding to issues well
+PD--Like some of the other bulleted lists, the bullets here are rather long. Can you pare down the content any more or try another presentation?
 
 ____
 "While the size and skill of the development community constrains the rate at which tickets can be resolved, the project should at least try to acknowledge each ticket the moment it appears.
 Even if the ticket lingers for a while, a response encourages the reporter to stay involved, because she feels that a human has registered what she has done (remember that filing a ticket usually involves more effort than, say, posting an email)."—Karl Fogel, Producing Open Source Software.footnote:[https://producingoss.com/en/producingoss-letter.pdf[_https://producingoss.com/en/producingoss-letter.pdf], page 64, accessed 24 June 2020 11:46 CET]
+PD--Would the quote here be better before the subhead?
```

Comment:
I have no thoughts on this and leave it to the book editors. We may have a style convention for such pull quotes.